README.md

```diff
@@ -0,0 +1,88 @@
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Please put here some general information about your Intergration / App / Solution.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
```

Comment:
Do we want to use this hacky way of scoling? 

If so, we should use #readme anchor

Reply:
All removed